### DISCUSSION TITLE
Recommended tool for output .prof file

### DISCUSSION THREAD

**Discussion by prachi30** [pythonspeed/filprofiler#512] [Q&A] [answered]
What is the recommended tool for visualizing .prof files outputted by FIL?

**Comment by itamarst** [comment, marked as answer]
Fil generates a flamegraph for you, so you can just open the `index.html` file to see it. However, if you want to visualize it yourself, you can use https://github.com/jonhoo/inferno/ or https://github.com/brendangregg/FlameGraph on the CLI, or upload to https://www.speedscope.app/
